src/VisualStudio/Core/Def/Implementation/ProjectSystem/VisualStudioWorkspace.cs

```diff
@@ -16,7 +17,7 @@ namespace Microsoft.VisualStudio.LanguageServices
     /// </summary>
     public abstract class VisualStudioWorkspace : Workspace
     {
-        private readonly BackgroundCompiler _backgroundCompiler;
+        private BackgroundCompiler _backgroundCompiler;
```

Comment:
A common counter-argument is that then we're holding onto the disposed object, but in theory the disposed object should have released it's resources...if it didn't then we don't understand our own code.